packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -351,14 +351,47 @@ export class MgtLogin extends MgtTemplatedComponent {
         light-dismiss
         @opened=${this.flyoutOpened}
         @closed=${this.flyoutClosed}>
-        <div slot="flyout">
-          <fluent-card class="flyout-card">
-            ${this.renderFlyoutContent()}
-          </fluent-card>
-        </div>
+        <fluent-card 
+          slot="flyout" 
+          tabindex="0" 
+          class="flyout-card"
+          @keydown=${this.onUserKeyDown}
+          >
+          ${this.renderFlyoutContent()}
+        </fluent-card>
       </mgt-flyout>`;
   }
 
+  /**
+   * Tracks tabbing through the flyout (keydown)
+   */
+  private readonly onUserKeyDown = (e: KeyboardEvent): void => {
+    if (!this.flyout.isOpen) {
+      return;
+    }
+
+    const el = this.renderRoot.querySelector('.popup-content');
+    const focusableEls = el.querySelectorAll('ul, fluent-button');
+    const firstFocusableEl = el.querySelector('#signout-button') || focusableEls[0];
+    const lastFocusableEl =
+      el.querySelector('#signin-different-account-button') || focusableEls[focusableEls.length - 1];
+
+    if (e.key === 'Tab' && e.shiftKey && firstFocusableEl === e.target) {
+      e.preventDefault();
+      (lastFocusableEl as HTMLElement)?.focus();
+    }
+    if (e.key === 'Tab' && lastFocusableEl === e.target) {
+      if (e.shiftKey) {
+        e.preventDefault();
+        const focusableArrs = Array.from(focusableEls);
+        window.setTimeout(() => (focusableEls[focusableArrs.indexOf(lastFocusableEl) - 1] as HTMLElement)?.focus(), 0);
+      } else {
+        e.preventDefault();
+        (firstFocusableEl as HTMLElement)?.focus();
+      }
+    }
```

Comment:
It seems to me that this could be simplified to:

```suggestion
    if (e.key === 'Tab' && !e.shiftKey && lastFocusableEl === e.target) {
      e.preventDefault();
      (firstFocusableEl as HTMLElement)?.focus();
    }
```

That way the browser handles the case where the shift key is pressed for us with native behaviour